akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/PresentationApp.scala

```diff
@@ -88,6 +88,12 @@ trait PresentationApp {
   }
 
   def handleResizeAndMoveSlide(msg: ResizeAndMoveSlide) {
+    // Force coordinate that are out-of-bounds inside valid values
+    val xOffset = if (msg.xOffset <= 0) msg.xOffset else 0
+    val yOffset = if (msg.yOffset <= 0) msg.yOffset else 0
+    val width = if (msg.widthRatio <= 100) msg.widthRatio else 100

```

Comment:
Here are client logs of page coordinates. x and y will always be 0 or less. 0 is the viewport point.
W and H will always be 100 or less. 100 means the page is displayed full page while less than 100 means the page is zoomed in.
=========

**** page x=0, y=0, w=100, h=100*****
**** page x=-1.0444855908465085, y=-0.9310503862561962, w=95.23809523809524, h=95.23809523809524*****
**** page x=-1.9940179461615153, y=-1.7774598283072864, w=90.9090909090909, h=90.9090909090909*****
**** page x=-2.860982270579566, y=-2.5502684493104564, w=86.95652173913044, h=86.95652173913044*****
**** page x=-3.6556995679627815, y=-3.258676351896695, w=83.33333333333333, h=83.33333333333333*****
**** page x=-4.3868394815553335, y=-3.9104116222760346, w=80, h=80*****
**** page x=-5.061737863333078, y=-4.5120134103185, w=76.92307692307692, h=76.92307692307693*****
**** page x=-5.686643772386544, y=-5.069052102950412, w=74.07407407407408, h=74.07407407407408*****
**** page x=-6.266913545079048, y=-5.586302317537191, w=71.42857142857142, h=71.42857142857142*****
**** page x=-6.807164712758274, y=-6.067880103531775, w=68.9655172413793, h=68.9655172413793*****
**** page x=-7.311399135925554, y=-6.517352703793386, w=66.66666666666666, h=66.66666666666666*****
**** page x=-7.783102305985268, y=-6.937827071780054, w=64.51612903225806, h=64.51612903225806*****
**** page x=-8.225324027916246, y=-7.332021791767557, w=62.5, h=62.5*****
**** page x=-8.640744433366566, y=-7.702325922664911, w=60.606060606060595, h=60.606060606060595*****
**** page x=-8.225324027916246, y=-7.332021791767557, w=62.5, h=62.5*****
**** page x=-7.783102305985268, y=-6.937827071780054, w=64.51612903225806, h=64.51612903225806*****
**** page x=-7.311399135925554, y=-6.517352703793386, w=66.66666666666666, h=66.66666666666666*****
**** page x=-6.807164712758274, y=-6.067880103531775, w=68.9655172413793, h=68.9655172413793*****
**** page x=-6.266913545079048, y=-5.586302317537191, w=71.42857142857142, h=71.42857142857142*****
**** page x=-5.6866437723865415, y=-5.069052102950412, w=74.07407407407408, h=74.07407407407408*****
**** page x=-5.061737863333074, y=-4.5120134103185, w=76.92307692307692, h=76.92307692307693*****
**** page x=-4.3868394815553335, y=-3.9104116222760346, w=80, h=80*****
**** page x=-3.6556995679627815, y=-3.258676351896695, w=83.33333333333333, h=83.33333333333333*****
**** page x=-2.860982270579566, y=-2.5502684493104564, w=86.95652173913044, h=86.95652173913044*****
**** page x=-1.9940179461615153, y=-1.7774598283072895, w=90.9090909090909, h=90.9090909090909*****
**** page x=-1.0444855908465085, y=-0.9310503862561994, w=95.23809523809524, h=95.23809523809524*****
**** page x=0, y=-3.4408849189351103e-15, w=100, h=100*****
**** page x=0, y=-3.4408849189351103e-15, w=100, h=100*****
**** page x=0, y=-3.4408849189351103e-15, w=100, h=100*****
**** page x=0, y=-3.4408849189351103e-15, w=100, h=100*****
**** page x=0, y=-3.4408849189351103e-15, w=100, h=100*****
**** page x=0, y=-3.4408849189351103e-15, w=100, h=100*****
**** page x=0, y=-3.4408849189351103e-15, w=100, h=100*****
**** page x=0, y=-3.4408849189351103e-15, w=100, h=100*****
**** page x=0, y=-3.4408849189351103e-15, w=100, h=100*****